Valet/VALSecureEnclaveValet.h

```diff
@@ -32,33 +29,24 @@ NS_CLASS_AVAILABLE_IOS(8_0)
 + (BOOL)supportsSecureEnclaveKeychainItems;
 
 /// Creates a Valet that reads/writes Secure Enclave keychain elements.
-- (nullable instancetype)initWithIdentifier:(NSString *)identifier;
+- (nullable instancetype)initWithIdentifier:(nonnull NSString *)identifier;
 
 /// Creates a Valet that reads/writes Secure Enclave keychain elements that can be shared across applications written by the same development team.
 /// @param sharedAccessGroupIdentifier This must correspond with the value for keychain-access-groups in your Entitlements file.
-- (nullable instancetype)initWithSharedAccessGroupIdentifier:(NSString *)sharedAccessGroupIdentifier;
+- (nullable instancetype)initWithSharedAccessGroupIdentifier:(nonnull NSString *)sharedAccessGroupIdentifier;
 
-/// Convenience method for inserting data into the keychain with a user prompt.
-/// @param userPrompt The prompt displayed to the user in Apple's Touch ID and passcode entry UI when updating a value.
-- (BOOL)setObject:(NSData *)value forKey:(NSString *)key userPrompt:(NSString *)userPrompt;
 /// Convenience method for retrieving data from the keychain with a user prompt.
```

Comment:
You're right that the README is a lot clearer on when this will return `nil` vs the docs here. I can update.

Summary: `objectForKey:` and `stringForKey` return `nil` whenever `canAccessKeychain` returns `NO`, or when the object doesn't exist in the keychain.